modules/05_add_blob_storage/src/blob_storage.py

```diff
@@ -0,0 +1,39 @@
+import os
+import uuid
+from azure.storage.blob import BlobServiceClient, ContentSettings
+
+class BlobStorage: # pylint: disable=too-few-public-methods
+    """
+        Blob store to store secret words.
+    """
+
+    def __init__(self):
+        self.account_name = os.getenv('AZURE_STORAGE_ACCOUNT_NAME', '')
+        self.account_key = os.getenv('AZURE_STORAGE_ACCOUNT_KEY', '')
+        connection_string = f"DefaultEndpointsProtocol=https;AccountName={self.account_name};AccountKey={self.account_key};EndpointSuffix=core.windows.net"
+
+        self.client = BlobServiceClient.from_connection_string(connection_string)
+        self.container_name = 'images-to-guess'
+        self.container_client = self.client.get_container_client(self.container_name)
+        try:
+            self.container_client.create_container(timeout=1000)
+        except Exception as exp: # pylint: disable=broad-except
```

Comment:
By the way. Maybe remove try except altogether? It's not an essential concept

Reply:
Done, I am gracefully checking first if it exists.